spot-client/src/spot-remote/ui/views/help.js

```diff
@@ -54,13 +52,8 @@ class Help extends React.Component {
                         <div className = 'help-message'>
                             This app is a remote controller for a conference room.
                             <div>
-                                Open&nbsp;
-                                <a
-                                    className = 'spot-home-link'
-                                    href = { spotTvHomeUrl } >
-                                    { spotTvHomeUrl }
-                                </a>
-                                &nbsp;in a browser to setup your conference room and obtain a share key.
+                                Open&nbsp;{ Help._getSpotTvUrl() }&nbsp;in a browser to setup your conference room
```

Comment:
The non-breaking spaces might be removable.

Reply:
yes